Gemfile

```diff
@@ -1,13 +1,15 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in cocoapods-xcautotest.gemspec
 gemspec
 
+# websocket server
+gem 'sinatra', '~> 1.4.6'
+
 group :development do
-  gem 'cocoapods'
+  gem 'cocoapods', '~> 1.0', '>= 1.0.1'
 
-  gem 'mocha'
-  gem 'bacon'
-  gem 'mocha-on-bacon'
-  gem 'prettybacon'
+  gem 'mocha', '~> 1.1'
+  gem 'bacon', '~> 1.2'
+  gem 'mocha-on-bacon', '~> 0.2.2'
+  gem 'prettybacon', '~> 0.0.2'
```

Comment:
All of these can be deleted if they're going in the Gemspec ( see Danger's for an example - https://github.com/danger/danger/blob/master/Gemfile )